app/src/tabs/Statistics/WordCloud.tsx

```diff
@@ -0,0 +1,45 @@
+import { styled } from "linaria/react";
+import React from "react";
+import AutoSizer from "react-virtualized-auto-sizer";
+import ReactWordcloud, { Options } from "react-wordcloud";
+
+import { color } from "@architus/facade/theme/color";
+
+export type WordData = {
+  text: string;
+  value: number;
+};
+
+type WordCloudProps = {
+  words: Array<WordData>;
+};
+
+const Styled = {
+  AutoSizer: styled(AutoSizer)`
+    width: 100%;
+  `,
+};
+export const WordCloud: React.FC<WordCloudProps> = React.memo(({ words }) => {
+  const options: Partial<Options> = {
+    rotations: 2,
+    rotationAngles: [-90, 0],
+    fontSizes: [5, 120],
+    enableOptimizations: true,
+    enableTooltip: false,
+    // fontFamily: "Renner*",
```

Comment:
Are these needed?

Reply:
I kinda left them there on purpose cause I'd like to work on this component more eventually